p2pd/main.go

```diff
@@ -22,6 +24,10 @@ func main() {
 	bootstrapPeers := flag.String("bootstrapPeers", "", "comma separated list of bootstrap peers; defaults to the IPFS DHT peers")
 	dht := flag.Bool("dht", false, "Enables the DHT in full node mode")
 	dhtClient := flag.Bool("dhtClient", false, "Enables the DHT in client mode")
+	connMgr := flag.Bool("connManager", false, "Enables the Connection Manager")
+	connMgrLo := flag.Int("connLo", 256, "Connection Manager Low Water mark")
+	connMgrHi := flag.Int("connHi", 512, "Connection Manager High Water mark")
+	connMgrGrace := flag.Int("connGrace", 120, "Connection Manager grace period (in seconds)")
```

Comment:
let's make this `flag.Duration` which can parse duration strings a la `1hr5min` or `5s`

Reply:
oh nice, didn't know about it.

Reply:
done.